homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -2,6 +2,8 @@
 from datetime import timedelta
 import logging
 
+from opendata_transport import OpendataTransport, exceptions
```

Comment:
We don't need to import `exceptions` it seems. We're only catching one exception from opendata_transport and that's `OpendataTransportError`.

Reply:
Good catch @MartinHjelmare! Solved it with https://github.com/home-assistant/home-assistant/pull/29228/commits/4c42c286041f4e4198332afe0cebd13ff0a0409e.
Thanks for reviewing my PRs :)